torch/csrc/jit/passes/utils/memory_dag.cpp

```diff
@@ -1,11 +1,25 @@
 #include "memory_dag.h"
 
+#include <c10/util/flat_hash_map.h>
 #include <torch/csrc/utils/memory.h>
 #include <algorithm>
 #include <queue>
 
 namespace torch {
 namespace jit {
+namespace {
+std::vector<const Element*> decomprMap;
```

Comment:
Let's use a more descriptive name, since now that we don't use "compression" to describe things anymore.